doc/hotspot-style.md

```diff
@@ -725,12 +725,17 @@ for further discussion.
 
 ### nullptr
 
-Prefer `nullptr`
+Use `nullptr`
 ([n2431](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2007/n2431.pdf))
-to `NULL`.  Don't use (constexpr or literal) 0 for pointers.
+rather than `NULL`.  Don't use (constant expression or literal) 0 for pointers.
```

Comment:
Should there be any discussion here of why we want to avoid `NULL`?  Specifically the varying
definitions and the pitfalls of `NULL` in varargs context.

Reply:
I think it would be enough to write 1..2 sentences about this, and then defer to N2431 already linked here for more details.

Reply:
I agree with Aleksey.

Reply:
Good point.  I decided just referring to the paper for rationale is sufficient.